### PR TITLE
Improve configuration error handling for Pivotal

### PR DESCRIPTION
**Pull request by shosti**

### CODE REVIEW

File: integrations/pivotal.rb [hunk 1]
```diff
@@ -22,11 +22,23 @@ def on_event(event)
...
+        rescue Http::Exceptions::HttpException => ex
+          case ex.response.code
+          when 403
+            msg = "#{ex.response.body['error']} Possible fix: #{ex.response.body['possible_fix']}"
+            raise ConfigurationError.new msg, original_exception: ex
+          when 400..499
+            raise ConfigurationError.new ex.response.body['error'], original_exception: ex
+          else
+            raise ex
+          end
```
Comment (nxvl) [inline integrations/pivotal.rb]: Can you please put this into a separate function so it's more readable? nested stuff is quite hard to read usually.

**Comment by nxvl** [comment]
looks good other than code readability a.k.a. me being pretty picky.

**Comment by shosti** [comment]
@nxvl better?

**Comment by nxvl** [comment]
yup :shipit:

**Pull request merged**